include/envoy/ratelimit/ratelimit.h

```diff
@@ -34,5 +39,48 @@ struct Descriptor {
   absl::optional<RateLimitOverride> limit_ = absl::nullopt;
```

Comment:
I think we could add optional token bucket here and simplify?

Reply:
We made LocalDescriptor and Descriptor separate to keep it in sync with what is in proto...